### PR TITLE
Experimental: do not take datetime into consideration

### PR DESCRIPTION
**Pull request by xlqian**
http://jira.canaltp.fr/browse/ITI-311

### CODE REVIEW

File: source/georef/street_network.cpp [hunk 1]
```diff
@@ -355,13 +355,7 @@ PathFinder::start_dijkstra_and_fill_duration_map(const navitia::time_duration& r
...
         } else {
             navitia::time_duration best_dist = bt::pos_infin;
-            if (distances[projection[source_e]] < bt::pos_infin) {
-                best_dist = distances[projection[source_e]] + crow_fly_duration(projection.distances[source_e]);
-            }
-            if (distances[projection[target_e]] < bt::pos_infin) {
-                best_dist = std::min(best_dist, distances[projection[target_e]]
-                                                          + crow_fly_duration(projection.distances[target_e]));
-            }
+            std::tie(best_dist, std::ignore) = find_nearest_vertex(projection, true);
```
Comment (TeXitoi) [inline source/georef/street_network.cpp]:
```c++
navitia::time_duration best_dist = find_nearest_vertex(projection, true).first;
```

File: source/jormungandr/tests/check_utils.py [hunk 2]
```diff
@@ -497,6 +501,8 @@ def is_valid_journey(journey, tester, query):
         g = s.get('geojson')
         g is None or shape(g)
 
+        assert (s['from']['id'] == previous_destination_uri)
+        previous_destination_uri = s['to']['id']
```
Comment (TeXitoi) [inline source/jormungandr/tests/check_utils.py:505]: Maybe do the same with departure/arrival datetime?

**Review by TeXitoi — COMMENTED** [review]
Great job!

File: source/jormungandr/tests/check_utils.py (hunk 2 above)
Comment (xlqian) [reply source/jormungandr/tests/check_utils.py:505]: it's tested at L496 https://github.com/CanalTP/navitia/pull/1915/files#diff-75dec1e2d23a2e9bc7163d480c7808edR496

File: source/georef/street_network.cpp (hunk 1 above)
Comment (xlqian) [reply source/georef/street_network.cpp]: done

**Review by TeXitoi — APPROVED** [review]

**Pull request merged**